src/site/paragraphs/table.drupal.liquid

```diff
@@ -1,57 +1,54 @@
-{% comment %}
-    Example data:
-    entity: {
-        "entityId": "3944",
-        "fieldTable": {
-            "tableValue": "Row 1 Column 1 Row 1 Column 2 Row 1 Column 3 Row 2 Column 1 Row 2 Column 2 Row 2 Column 3 Row 3 Column 1 Row 3 Column 2 Row 3 Column 3",
-            "value": {
-                "0": [
-                "Row 1 Column 1",
-                "Row 1 Column 2",
-                "Row 1 Column 3"
-                ],
-                "1": [
-                "Row 2 Column 1",
-                "Row 2 Column 2",
-                "Row 2 Column 3"
-                ],
-                "2": [
-                "Row 3 Column 1",
-                "Row 3 Column 2",
-                "Row 3 Column 3"
-                ],
-            }
-            "format": "rich_text",
-            "caption": "Test Table"
-        }
-    }
-{% endcomment %}
+<!-- Mobile -->
+<div class="vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-u-display--none"
+  data-template="paragraphs/table-mobile">
+  {% assign mobileTableData = entity | formatMobileTableData %}
+  <table class="usa-table-borderless">
```

Comment:
Might be good to get @jenstrickland's insight into this table markup 🙂

Reply:
Thanks, @kelsonic ! 

- The table markup for desktop and mobile must be identical. All layout changes must happen in the CSS. 

- The table markup must include the roles as shown in the Codepen, because as soon as CSS is applied to make the table responsive, the semantics of the table will be removed. The roles retain the semantics.

- The table markup must include the scope attribute as shown in the Codepen.

- The table markup should use a `th` for the first cell of each row, as it is a heading for the row.

- I'm not sure why `p` and `strong` is being used here. The `th` and `dfn` elements would be more semantic.